README.md

```diff
@@ -228,13 +228,31 @@ All Databricks resources bypass policy enforcement at creation time. Databricks
 
 ### Resources that are exempt from policy evaluation
```

Comment:
should also add a section about types that are marked 'internal' in operations api - eg microsoft.devtestlab/schedules

Reply:
@varunkumta - can you point me to where I can find these?